service/README.md

```diff
@@ -84,6 +84,11 @@ skaffold build --file-output artifacts.json
 skaffold test --build-artifacts artifacts.json
 ```
 
+### Pre-push checklist
+
+- From `/service`, run `cargo fmt` and `cargo clippy --all-targets --all-features -- -D warnings` before pushing.
```

Comment:
this should probably reference `just`. Update this section or remove/reference to a doc that is more complete